app/src/main/kotlin/com/antosik/benchproject/app/more/view/layout/MoreScreen.kt

```diff
@@ -1,4 +1,4 @@
-package com.antosik.benchproject.app.more.view
+package com.antosik.benchproject.app.more.view.layout
```

Comment:
Is `layout` package really needed?

Reply:
I thought about it and I tried somehow divide this package but it could be just in view because this jetpack compose view is directly associated with `MoreFragemnt` therefore it shouldn't be in another place like common `layout/view` 😄

Reply:
I left it in view